plugins/greenhouse/src/data.ts

```diff
@@ -172,13 +176,13 @@ async function getItems(
                 case "date":
                 case "link":
                     fieldData[field.id] = {
-                        value: value ? String(value) : null,
+                        value: v.is(StringifiableSchema, value) ? String(value) : null,
                         type: field.type,
                     }
                     break
                 case "multiCollectionReference": {
-                    const ids: string[] = []
-                    if (Array.isArray(value)) {
+                    const ids = []
+                    if (v.is(ArrayWithIdsSchema, value)) {
```

Comment:
I guess it's this that isn't quite right

Reply:
Good catch, the ids are numbers, not strings 👌